### PR TITLE
Add codeowners

### PR DESCRIPTION
**Pull request by e00E**
https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners
This automatically requests reviews from the team for non draft PRs and allows us to enforce that PR approvals come from the team.

### CODE REVIEW

**Review by josojo — APPROVED** [review]
I am a code owner ;) yeah!

**Comment by e00E** [comment]
Will merge since it's uncontroversial and so that I can test that it works.

**Comment by fleupold** [comment]
Collective code ownership for the win! I'm not sure if we need to enforce that approvals come from the team (I'd trust everyone in the company to only merge reasonable changes or ask for proper review) but the auto request review feature is very useful...

**Pull request merged**
